sdk/communication/Azure.Communication.Chat/src/Generated/ChatThreadRestClient.cs

```diff
@@ -753,40 +754,49 @@ public Response<ChatParticipantsCollection> ListChatParticipants(string chatThre
             }
         }
 
-        internal HttpMessage CreateRemoveChatParticipantRequest(string chatThreadId, string chatParticipantId)
+        internal HttpMessage CreateRemoveChatParticipantRequest(string chatThreadId, string rawId, CommunicationUserIdentifierModel communicationUser, PhoneNumberIdentifierModel phoneNumber, MicrosoftTeamsUserIdentifierModel microsoftTeamsUser)
```

Comment:
Is this autogenerated ? raw Id is technically part of the CommunicationUserIdentifierModel ..

Reply:
Yes, this is auto-generated. And yes it looks a bit strange to me too, I think we could just pass in a CommunicationIdentifierModel instead of all of these: string rawId, CommunicationUserIdentifierModel communicationUser, PhoneNumberIdentifierModel phoneNumber, MicrosoftTeamsUserIdentifierModel microsoftTeamsUser

Reply:
This is being flattened by Autorest, considering that it is an internal method, it should be fine